exalearn-ui/src/app/pages/test-page/test-page.component.ts

```diff
@@ -10,9 +12,11 @@ import { Question } from '../../interfaces/interfaces';
 export class TestPageComponent implements OnInit {
 	testQuestions: Question[] = [];
 
-	constructor() {}
+	constructor(public translateService: TranslateService) {}
 
 	ngOnInit() {
 		this.testQuestions = questions;
+		//TODO use user state from backend
```

Comment:
Expand a comment a bit. Describe in short what can happen if we remove this line.